### ISSUE TITLE
scapholunate ligament tear

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10725]
**Preferred term label:**
scapholunate ligament tear

**Synonyms**

**Definition (free text, please give PubMed ID)**
Advanced imaging revealing a tear in any part of the scapholunate interosseous ligament

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Scapholunate (SL) ligament injury

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I am interpreting this as a complete tear. If this is not the case please open a new issue

Adding new term: 
- ID: HP:6001206
- label: Scapholunate ligament rupture
- parents: [Abnormal carpal morphology (HP:0001191)](https://hpo.jax.org/app/browse/term/HP:0001191)
- pmid: PMID:32684689
- def: A complete breakage of the complete rupture of the scapholunate ligament. The most common cause is physical trauma to the wrist such as a fall onto an outstretched hand. Rupture or tear of the ligmanet leads to tear of the scapholunate ligament leads to carpal instability.
- comment: Together with the lunotriquetral interosseous ligament, the scapholunate ligament is one of the two proximal interosseous carpal ligaments of the wrist. It is a C-shaped ligament that binds the scaphoid bone to the lunate bone of the wrist.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10725
- synonyms:

**Comment by MickeySegal** [comment]
At the minimum inclusion of the parent term in the Definition is worded confusingly.  

I've asked the orthopedist who created this term: "Should “Scapholunate ligament tear” be renamed to rupture or broadened to include rupture?  Or should we push back and ask for a finding of tear only?"

**Comment by MickeySegal** [comment]
The orthopedist replied:

"A scapholunate tear can be partial or complete. Since it is "C" shaped, it often is only one limb of the "C" that is torn. I am okay using the term rupture and tear interchangeably in this case. "

From this it sounds like tear is the best choice, with partial or complete/rupture being part of the synonyms or definition.

**Comment by pnrobinson** [comment]
reopening to check

**Comment by pnrobinson** [comment]
@MickeySegal 
Changed the label to  "Scapholunate ligament tear" and put "rupture" as RELATED synonym. Also added this as synonym: Scapholunate ligament injury
